pages/vi/vi-github-and-repositories.md

```diff
@@ -27,7 +27,8 @@ This is just a summary of the steps that you will need to perform. Please, keep
 * [Sync Your Fork](#Sync_Your_Fork)
 
 
-** Note :** You will see some common names used in git (*which you can change*), such as `master: the name of the main branch`, `upstream: the location where you forked the repository from`, and `origin: the location you cloned repository from`. Both `upstream and origin` are considered **remote**. Keep in mind also, a repository may contain many branches.
+**NOTE**: You will see some common names used in git (*which you can 
```

Comment:
you break line so please fix this

Reply:
I believe you mean that I need to delete line break above the change?

Reply:
It does show up well on the link (raw.githack)

Reply:
@lmmrssa Ah, sorry for some reason I didn't see that break. I thought you meant the one above the note.